tests/functional/test_install_reqs.py

```diff
@@ -171,7 +171,7 @@ def test_install_local_editable_with_extras(script, data):
 @pytest.mark.network
 def test_install_collected_dependencies_first(script):
     result = script.pip(
-        'install', 'paramiko',
+        'install', 'paramiko==1.17',
```

Comment:
Instead of needing paramiko, we could perform this test without the need for network with `toporequires2` which requires `toporequires` (with ``--no-index`, '-f', data.find_links`)